src/main/java/com/google/cloud/tools/appengine/cloudsdk/Gcloud.java

```diff
@@ -102,6 +103,25 @@ public CloudSdkGenRepoInfoFile newGenRepoInfo(ProcessHandler processHandler) {
     return CloudSdkComponent.fromJsonList(componentsJson);
   }
 
+  /**
+   * Returns a representation of gcloud config, it makes a synchronous call to gcloud config list to
+   * do so.
+   */
+  public CloudSdkConfig getConfig()
+      throws CloudSdkNotFoundException, CloudSdkOutOfDateException, CloudSdkVersionFileException,
+          IOException, ProcessHandlerException {
+    sdk.validateCloudSdk();
+
+    List<String> command =
+        new ImmutableList.Builder<String>()
```

Comment:
We can just do `gcloud config get-value project`, but I'm guessing you'll need other values from the entire config info?

Reply:
Yeah at some point.